### PR TITLE
feat(start): Add initialPath param to config

### PR DESCRIPTION
**Pull request by georgespyropoulos**
Allow an initial path to be set so a specific route can be loaded when running `sku start`

EXAMPLE USAGE:

`sku.config.js`

```js
module.exports = {
  entry: {
   ...
  initialPath: '/my-page'
  ...
};
```

### CODE REVIEW

File: scripts/start-ssr.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const copyPublicFiles = () => {
 runWebpack(webpackConfig[1])
   .then(copyPublicFiles)
   .then(() => {
-    const url = `http://${hosts[0]}:${port.backend}`;
+    const url = `http://${hosts[0]}:${port.backend}${initialPath}`;
```
Comment (jahredhope) [inline scripts/start-ssr.js]: Should this use a [url joining library](https://www.npmjs.com/package/url-join) to handle missing/duplicate slashes?

**Comment by markdalgleish** [comment]
Not sure a CLI argument makes sense for this one, since this is a config value that might differ for every project in a monorepo setup, e.g.

```js
module.exports = [
  {
    ...,
    name: 'my-first-page',
    initialPath: '/my-first-page'
  },
  {
    ...,
    name: 'my-second-page',
    initialPath: '/my-second-page'
  }
];
```

**Comment by georgespyropoulos** [comment]
I still prefer having this option in `sku.config.js` but the consensus at a FEED meeting was to have it as a CLI argument. 

I think the argument against having it in the config was that people felt that SKU should have minimal config and the initial path could easily be set in a package.json script. 

I think that a CLI argument fragments the configuration options for SKU. They're no longer all together in the sku config.

**Comment by hscgavin** [comment]
`builds.ssr.js` has been forgotten? 🤔

**Pull request merged**